.travis.yml

```diff
@@ -13,6 +13,8 @@ after_script: npm run publish-coverage
 cache:
   directories:
   - node_modules
+before_deploy:
+  - npm run build
```

Comment:
Building should be part of the test run in CI so that if the build fails, the test run fails.